README.md

```diff
@@ -1,18 +1,14 @@
 ### Under Construction
 
-#### This repo is under heavy, active revision. The code does not work yet. Browse, but don't judge. :)
+#### This repo is under active revision. The code runs but is not yet complete, so please don't judge. :)
```

Comment:
I would prefer to say "does not work yet" until we feel confident it actually does work well.